ptvsd/wrapper.py

```diff
@@ -2434,6 +2452,13 @@ def on_pydevd_thread_suspend(self, seq, args):
             exc_name, exc_desc, _, _  = \
                 self._parse_exception_details(resp_args, include_stack=False)
 
+        if self.is_vs_client():
+            if self._vs_ignore_stopped_events.isSet() and reason == 'pause':
```

Comment:
This should be `is_set` (`isSet` is the older deprecated spelling, and I think it's already gone in 3.x?).